### ISSUE TITLE
test_polynomial.TestDocs overly sensitive to output formatting (migrated from Trac #1460)

### ISSUE THREAD

**Issue by thouis** [thouis/numpy-trac-migration#3012]
Original ticket http://projects.scipy.org/numpy/ticket/1460
Reported 2010-04-23 by atmention:drnlm, assigned to unknown.

Seen on the windows buildbot.

Using python 2.5, Windows XP, Mingw32 build.

```
FAIL: test_doctests (test_polynomial.TestDocs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\buildbot\numpy\b11\numpy-install25\Lib\site-packages\numpy\lib\tests\test_polynomial.py", line 90, in test_doctests
    return rundocs()
  File "..\numpy-install25\Lib\site-packages\numpy\testing\utils.py", line 962, in rundocs
AssertionError: Some doctests failed:
**********************************************************************
File "C:\buildbot\numpy\b11\numpy-install25\Lib\site-packages\numpy\lib\tests\test_polynomial.py", line 20, in test_polynomial
Failed example:
    print(poly1d([100e-90, 1.234567e-9j+3, -1234.999e8]))
Expected:
           2
    1e-88 x + (3 + 1.235e-09j) x - 1.235e+11
Got:
            2
    1e-088 x + (3 + 1.235e-009j) x - 1.235e+011
```

Not seen using python 2.6

**Comment by thouis** [comment]
Comment in Trac by atmention:drnlm, 2010-04-23

**Comment by thouis** [comment]
Comment in Trac by atmention:rgommers, 2011-03-01

Was fixed in commit:f4d9da1e
